### PR TITLE
[WIP] Add ddmrp from Eficent repository

### PR DESCRIPTION
**Pull request by guewen**
* Extract the addon from https://github.com/Eficent/ddmrp
* Migrate ddmrp to version 11.0
* Note: working on https://github.com/Eficent/ddmrp/pull/47 until ready, then I'll update this one for inclusion in OCA

The other addons of the repository will need to be extracted the same way.

Require #2 for the build

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
Thanks @guewen We're now busy preparing the training for next week, and we'll support you next week.
Please consider migrating this as well: https://github.com/Eficent/ddmrp/commit/a88a005d2e3aae1a4fd664243e58ce691dc94b21

**Comment by guewen** [comment]
I set this PR as `Work in progress`, I'll continue the work on https://github.com/Eficent/ddmrp/pull/47 and update this one when ready. It'll ease the port of changes from other branches if it happens in the same repo.

**Comment by pedrobaeza** [comment]
Please rebase to have CI and you can move WIP branch here for better tracking to all contributors.

**Comment by LoisRForgeFlow** [comment]
@guewen there are some dependencies that aren't even proposed on a PR to v11 (as stock_demand_estimate), how are you working around that?

**Comment by guewen** [comment]
@lreficent It is: https://github.com/OCA/stock-logistics-warehouse/pull/381 :)

**Comment by LoisRForgeFlow** [comment]
@guewen yes, finally I found it (it has a name without underscores). I'm reviewing all the dependencies on OCA for ddmrp, I hope we could get everything merged ASAP and then move to review ddmrp itself! (the general overview is good :wink:)

**Comment by LoisRForgeFlow** [comment]
@guewen Everything is merged and ready at Eficent repo, would you update this? :wink:

**Comment by max3903** [comment]
@guewen @lreficent Can we wait for https://github.com/Eficent/ddmrp/pull/60 to be merged?

**Comment by LoisRForgeFlow** [comment]
@max3903 Yes, as you prefer :+1:

File: ddmrp/README.rst [hunk 1]
```diff
@@ -0,0 +1,242 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
```
Comment (simahawk) [inline ddmrp/README.rst:1]: maybe switch to new readme structure?

**Review by simahawk — APPROVED** [review]
not reviewing everything since it has been already validated on Eficent repo, so I trust it's working.
Only remark: readme update if possible.

File: ddmrp/README.rst (hunk 1 above)
Comment (LoisRForgeFlow) [reply ddmrp/README.rst:1]: We are still not ready to move to OCA unfortunately. We are still working on some important improvements you can follow here: https://github.com/Eficent/ddmrp/pull/69. But for sure, when the code is ready I will update the readme!

**Review by max3903 — APPROVED** [review]

File: ddmrp/README.rst (hunk 1 above)
Comment (max3903) [reply ddmrp/README.rst:1]: @lreficent Can we merge this one and when you are ready, you make your pull request with the corresponding version number?

**Comment by LoisRForgeFlow** [comment]
Closed in favour of https://github.com/OCA/ddmrp/pull/16. Look at the description for more details.

**Pull request closed without merging**